src/server/supplements/har/harTracer.ts

```diff
@@ -235,20 +235,22 @@ export class HarTracer {
     const timing = response.timing();
     if (pageEntry.startedDateTime.valueOf() > timing.startTime)
       pageEntry.startedDateTime = new Date(timing.startTime);
+    const blocked = timing.startTime !== -1 ? helper.millisToRoundishMillis(timing.startTime - request.issueTime()) : -1;
```

Comment:
Need to change this to `secondsToMillis` once we agree on the solution here - https://github.com/microsoft/playwright/pull/7686/files#r671585936